platforms/core-configuration/input-tracking/src/test/groovy/org/gradle/internal/configuration/inputs/AccessTrackingPropertiesNonStringTest.groovy

```diff
@@ -129,10 +129,11 @@ class AccessTrackingPropertiesNonStringTest extends Specification {
 
     def "entrySet() enumeration is tracked for non-strings"() {
         when:
-        def result = ImmutableSet.copyOf(getMapUnderTestToRead().entrySet())
+        ImmutableSet.Builder<Map.Entry<Object, Object>> iterated = ImmutableSet.builder()
```

Comment:
Curious what caused this change

Reply:
The old implementation of `copyOf` used to [not call `isEmpty()`](https://github.com/google/guava/blob/v32.1.3/guava/src/com/google/common/collect/ImmutableSet.java#L225-L247), this [happens now](https://github.com/google/guava/blob/v33.4.5/guava/src/com/google/common/collect/ImmutableSet.java#L176-L206) and makes it two "enumerations" to the `AccessTrackingProperties`.